docs/features/series.md

```diff
@@ -91,26 +89,25 @@ GET playlist/xdAqW8ya
 }
 ```
 
-## Native series
+## (NEW) Native series
 
 JW Player has native series management from the JW Dashboard:
 
 - simplifies the series creation workflow
 - automatically calculate the number of episodes and duration of series
 - contains trailers and bonus content
 
-This section describes how this will work. 
+This section describes how this will work.
 
 ### Creating native series in the dashboard
 
-1. Customers define series 
-2. Customers create media items each of which represents a series
-3. Customers add a `seriesId` custom param to created media items
-4. Customers publish their series by putting media items into a playlist
+1. Customers create media items from the Media Library page with "Series" content type.
```

Comment:
I think it's probably safer / more maintainable if we put a link to the dashboard documentation for series when it's available. Trying to summarize the whole mechanism in a few bulletpoints w/o visuals is going to be tough. Probably fine to add a TODO here for this for now.

Reply:
Do you mean adding "TODO" for the whole section of the new flow? Or just for "Creating native series in the dashboard" part?

Reply:
Just the dashboard part. I think it's safer if we avoid adding too much dashboard documentation into this repo. The TODO would basically be:
```[//]: # (TODO: Add link)
See the series documentation for the JW dashboard.```